src/System.Text.Encodings.Web/tests/PerformanceTests.cs

```diff
@@ -0,0 +1,170 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.Diagnostics;
+using System.IO;
+using System.Runtime.CompilerServices;
+using System.Text.Encodings.Web;
+using Xunit;
+using Xunit.Abstractions;
+
+#if RELEASE 
```

Comment:
I did not do it because they take long. I did that so people don't compare debug perf results :-)